src/routes/user/userComponents/userData.svelte

```diff
@@ -26,35 +168,126 @@
                                     class="form--disabled input variant-form-material"
                                     type="text"
                                     name="inputField"
-                                    placeholder={userInfo[i]}
-                                    id="input_{key}"
+                                    placeholder={personalData.lastName}
+                                    bind:this={lastNameReference}
+                                />
+                            </label>
+                        </form>
+                    </td>
+                </tr>
+                <tr>
+                    <th class="text-left"> Telefonnummer: </th>
```

Comment:
```suggestion
                    <th class="text-left">Telefonnummer:</th>
```
Oder waren die Spaces da mit Absicht?